README.md

```diff
@@ -579,6 +579,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JUnit](http://junit.org/) - Common testing framework.
 * [JUnitParams](https://pragmatists.github.io/JUnitParams/) - Creation of readable and maintainable parametrised tests.
 * [Mockito](https://github.com/mockito/mockito) - Creation of test double objects in automated unit tests for the purpose of TDD or BDD.
+* [PIT](http://pitest.org) - Mutation-testing framework for Java and JVM.
```

Comment:
@akullpp check the recent commits. I have provided a more specific description for PIT and its applicability in testing. Do let me know if you have more suggestions.